<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/DeathA2/Downloads/warning.png</normaloff>C:/Users/DeathA2/Downloads/warning.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel
{
	image: url(:/new/image/img/Picture1.png);
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
	font-size: 15px;
	width: fit;
	font-style: bold;
}
QGroupBox{
background-color: rgb(184, 184, 184,220);
}
</string>
   </property>
   <property name="title">
    <string>Sign Up</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string>Tên người dùng:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(207, 207, 207);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userAccount">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(207, 207, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string>Tên tài khoản:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(207, 207, 207);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string>Mật khẩu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rePass">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(207, 207, 207);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string>Nhập lại mật khẩu:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPass">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/new/icon/img/unhide.png</normaloff>:/new/icon/img/unhide.png</iconset>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRePass">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/image/img/Picture1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/new/icon/img/unhide.png</normaloff>:/new/icon/img/unhide.png</iconset>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="errorPass">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(179, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mật khẩu phải có ít nhất 8 ký tự</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorUserAcc">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(179, 0, 0);</string>
    </property>
    <property name="text">
     <string>Tên tài khoản đã tồn tại</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorRePas">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(179, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mật khẩu không trùng khớp</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorUserName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(179, 0, 0);</string>
    </property>
    <property name="text">
     <string>Tên người dùng đã tồn tại</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/new/image/img/Picture1.png);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/new/image/img/Picture1.png);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/image/img/BackgroundNoiseStudy.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>groupBox</zorder>
  <zorder>Save</zorder>
  <zorder>Close</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
